modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/decisions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+    module V0
+        class DecisionsController < ApplicationController
+            before_action :set_power_of_attorney_request, only: [:show, :create]
+
+            def create
```

Comment:
Guessing what the most streamlined expression would be:
```ruby
reason = decision_params.delete(:reason)
reason = nil if decision_params[:type] == PowerOfAttorneyRequestDecision::Types::ACCEPTANCE

power_of_attorney_request.create_resolution!(
  resolving: PowerOfAttorneyRequestDecision.new(decision_params),
  creator: current_user,
  reason:
)
```

Reply:
It didn't want to delete reason out of decision params and if it's left in, it was breaking. creator also goes in decision so I had to move things around but this works using creation params

 creation_params = decision_params.merge(creator: current_user)
reason = creation_params.delete(:reason)
reason = nil if creation_params[:type] == PowerOfAttorneyRequestDecision::Types::ACCEPTANCE

@power_of_attorney_request.create_resolution!( 
            resolving: PowerOfAttorneyRequestDecision.new(creation_params), 
            reason:
)